extensions/amp-story/1.0/amp-story.js

```diff
@@ -1285,7 +1296,8 @@ export class AmpStory extends AMP.BaseElement {
     const uiState = this.getUIType_();
     this.storeService_.dispatch(Action.TOGGLE_UI, uiState);
 
-    if (uiState !== UIType.MOBILE) {
+    if (uiState !== UIType.MOBILE ||
+        this.getSupportedOrientations_().includes('landscape')) {
```

Comment:
is this right? shouldn't it be `!this.getSupportedOrientations_().includes('landscape')`?

Reply:
Ah, this is confusing but is actually correct.
What's confusing is the `TOGGLE_LANDSCAPE, false` right under it, that actually means `toggle the 'story only supports portrait' UI to false`.
We should rename this one, I filed #19670 and will do it in a follow up PR, thanks for raising the issue

Reply:
Didn't mean to resolve this... 😥